src/styles/main.scss

```diff
@@ -0,0 +1,40 @@
+html {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+body {
+  margin: 0 20px;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.calendar {
+  width: 7 * ($dayWidth + 1);
+  gap: $gap;
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: start;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    background-color: #eee;
+    gap: $gap;
+    width: $dayWidth;
+    height: 100px;
+    border: 1px solid black;
+  }
+
+  &__day:hover {
+    cursor: pointer;
+    transition-duration: 0.5s;
+    background-color: #ffbfcb;
+    transform: translateY(-20px);
+  }
```

Comment:
Try to fix animation. Need to be done more smoothly.
https://gyazo.com/7ca7bf437800202b5c932e603e8e7bdf